<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="recvGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>371</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>接收</string>
    </property>
    <widget class="QTextEdit" name="recvText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>371</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>191</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>网络设置</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>本机IP地址</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="ipData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>本机端口号</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="portData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="connectBtn">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>160</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>监听</string>
     </property>
    </widget>
    <widget class="QComboBox" name="clientList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="clearBtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>260</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>621</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>发送窗口</string>
    </property>
    <widget class="QTextEdit" name="sendData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>521</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sendBtn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>71</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Server</string>
    </property>
   </widget>
   <addaction name="menuServer"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
